src/app/ipv/router.js

```diff
@@ -48,6 +49,19 @@ router.post(
   csrfProtection,
   handleCriEscapeAction
 );
+router.post(
+  "/page/pyi-suggest-other-options",
+  parseForm,
+  csrfProtection,
+  handleCimitEscapeAction
+);
+router.post(
```

Comment:
You dont need this for this page, this can be handled with the next/end events. If you take a look at pyi-escape-page & pyi-escape-no-f2f it will make a bit more sense. We can handle a next & end events in core-back as standard. With the other page where its handling more than those two possibilities it requires the code you've added

Reply:
That makes sense - I've removed this code and also reverted the change to the nunjucks file for `pyi-suggest-other-options-no-f2f.njk`